plugins/plugin-java-debugger/che-plugin-java-debugger-server/src/test/resources/docker/entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2017 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+
+javac -g org/eclipse/ThreadDumpTest1.java
+javac -g org/eclipse/ThreadDumpTest2.java
+javac -g org/eclipse/StackFrameDumpTest1.java
+javac -g org/eclipse/GetValueTest1.java
+javac -g org/eclipse/EvaluateExpressionTest1.java
+javac -g com/HelloWorld.java
+
+java -Xdebug -Xrunjdwp:transport=dt_socket,address=8001,server=y,suspend=y org.eclipse.ThreadDumpTest1
```

Comment:
Maybe move common options into a variable?

Reply:
I meant create a variable with `-Xdebug -Xrunjdwp:transport=dt_socket,address=8001,server=y,suspend=y` and then use, for example:
`java $args org.eclipse.ThreadDumpTest1`
Easier to read.